src/contracts/examples/redstone-oev/RedstoneOevDappControl.sol

```diff
@@ -3,45 +3,58 @@ pragma solidity 0.8.25;
 
 import { DAppControl } from "src/contracts/dapp/DAppControl.sol";
 import { CallConfig } from "src/contracts/types/ConfigTypes.sol";
+import { SafeTransferLib } from "solady/utils/SafeTransferLib.sol";
+import { IRedstoneAdapter } from
+    "lib/redstone-oracles-monorepo/packages/on-chain-relayer/contracts/core/IRedstoneAdapter.sol";
+
 import "src/contracts/types/UserOperation.sol";
 import "src/contracts/types/SolverOperation.sol";
-import "./RedstoneAdapterAtlasWrapper.sol";
-import { SafeTransferLib } from "solady/utils/SafeTransferLib.sol";
 
 contract RedstoneOevDappControl is DAppControl {
-    error FailedToAllocateOEV();
     error OnlyGovernance();
     error OnlyWhitelistedBundlerAllowed();
+    error OnlyWhitelistedOracleAllowed();
+    error InvalidUserDestination();
+    error InvalidUserEntryCall();
+    error InvalidUserUpdateCall();
     error OracleUpdateFailed();
+    error InvalidOevShare();
+    error InvalidOevAllocationDestination();
 
-    uint256 public bundlerOEVPercent;
-    uint256 public atlasOEVPercent;
-    address public atlasVault;
-    mapping(address oracle => address oracleVault) public oracleVaults;
+    uint256 public constant OEV_SHARE_SCALE = 10_000;
+
+    // OEV shares are in hundredth of percent
+    uint256 public oevShareBundler;
+    uint256 public oevShareFastlane;
+
+    address public oevAllocationDestination;
 
     mapping(address bundler => bool isWhitelisted) public bundlerWhitelist;
     uint32 public NUM_WHITELISTED_BUNDLERS = 0;
 
+    mapping(address oracle => bool isWhitelisted) public oracleWhitelist;
+    uint32 public NUM_WHITELISTED_ORACLES = 0;
```

Comment:
Could name using camel case as all caps kinda implies its a constant, but it is not.